File: app/src/main/java/geert/berkers/soarcast/model/Locatie.java
package geert.berkers.soarcast.model;

/**
 * Created by Zorgkluis (Geert Berkers)
 */
public class Locatie {

    public Integer id;
    public Double lat;
    public Double lon;
    public Integer maxdeg;
    public Integer mindeg;
    public String naam;

    public Locatie(final String naam, final Integer id, final Double lat, final Double lon, final Integer mindeg, final Integer maxdeg) {
        this.naam = naam;
        this.id = id;
        this.lat = lat;
        this.lon = lon;
        this.mindeg = mindeg;
        this.maxdeg = maxdeg;
    }
}

File: app/src/main/java/geert/berkers/soarcast/Locatie.kt
package geert.berkers.soarcast

/**
 * Created by Zorgkluis (Geert Berkers)
 */
data class Locatie(
    var naam: String,
    var id: Int,
    var lat: Double,
    var lon: Double,
    var mindeg: Int,
    var maxdeg: Int
)